### DISCUSSION TITLE
Admob Integration

### DISCUSSION THREAD

**Discussion by jonathanroze** [invertase/react-native-firebase#5539] [General]
Hi, 

I Saw that Admob was supported by React-Native-Firebase in previous version but it's not supported anymore after V12.

Did you planned to implement it again ? I think I can use previous version (11.5) but I can't see any infos about SKNework and new tracking system for IOS 14.5) Can you help me ? 

Thank :)

**Comment by mikehardy** [comment]
#5329 - follow along there - it's on the task list but hasn't made it to top priority yet unforutnately
